http/http/src/main/java/io/helidon/http/RequestedUriDiscoveryContext.java

```diff
@@ -260,7 +260,7 @@ private void prepareAndCheckRequestedUriSettings() {
             }
             if (trustedProxies == null) {
                 trustedProxies = AllowList.builder()
-                        .addDenied(s -> true)
+                        .addDeniedPredicate(s -> true)
```

Comment:
seems cleaner the old way. Typically `Predicate` is in the class name, not in method names.

Reply:
It cannot be done the old way, because there is `denied` already which is the exact string match, and we cannot use the same method for two different return types

Reply:
Trying to "hack" it by using `@Singular` with a duplicate method name

Reply:
yes, that seems to work